component/src/main/java/org/wso2/carbon/identity/authenticator/reddit/RedditAuthenticator.java

```diff
@@ -128,7 +129,29 @@ public String getScope(String scope, Map<String, String> authenticatorProperties
      * @return weather true or false
      */
     public boolean canHandle(HttpServletRequest httpServletRequest) {
-        return httpServletRequest.getParameter(RedditAuthenticatorConstants.CODE) != null;
+        return httpServletRequest.getParameter(RedditAuthenticatorConstants.CODE) != null || httpServletRequest
+                .getParameter(RedditAuthenticatorConstants.OAUTH2_PARAM_ERROR)!=null;
```

Comment:
Format the code.

Reply:
Fixed in 712cd7b17f204023cc7106222b39e945853f010f